.fixtures.yml

```diff
@@ -6,7 +6,7 @@ fixtures:
     package: "puppetlabs/package"
     reboot: "puppetlabs/reboot"
     node_manager:
-      repo: "WhatsARanjit/node_manager"
+      repo: "https://github.com/puppetlabs/puppetlabs-node_manager.git"
       ref: "0.7.5"
```

Comment:
while you're at it, can we get a 1.0.1 release that includes https://github.com/puppetlabs/puppetlabs-node_manager/pull/88? And IMO the fixtures should not reference forge releases, it should use the HEAD branch of the modules, to catch breaking changes before they are released. the acceptance tests should use actual forge releases.

Reply:
I'm not able to make releases of node_manager unfortunately. I'll unpin the ref'd version soon, just need to test the latest release.

Reply:
Doesn't it make sense for the CAT team to make a new release? Also the latest release is 1.0.0 (but that's a bit odd https://github.com/puppetlabs/puppetlabs-node_manager/issues/86)

Reply:
I'll roll with 1.0.1 when CAT team is happy to release. I agree the release seems odd. I'd expected forge links to update.